### ISSUE TITLE
[Suggestion] Brick sized connectors OR ability to place them on child entities

### ISSUE THREAD

**Issue by Kaiser-Indrasil** [tsunamayo/Starship-EVO#2500]
I've been working on a starship replica which will feature 4 docking ports you will be able to connect various modules to, for example an escape pod, docking collar, scanner probes mini-hangar etc.

The problem I'm facing is, the connectors I place end up misaligned in relation to the original model. I would've easily countered that by placing the connectors on a grid that's offset by a rail but they cannot be placed on child entities, either.

This will be a 67m-long model, so with the amount of detail I've crammed onto this thing over the course of the last month (working on it full-time, no less), every offset is clearly noticeable and affects neighbouring areas of detailing.
![image](https://user-images.githubusercontent.com/63756057/85302939-b9e8d680-b4a1-11ea-9a4f-dc908b516b2a.png)

@edit: brick-sized docking port would also enable all kinds of modularity accross a wide variety of uses. For example - hardpoints with swappable turrets. Small ship escape pods (remember Lando's Millennium Falcon's escape module?). Drones and fighters being able to dock up with their mothership. Rearm and refuel vessels docking to fighters on the battlefield. And more

**Comment by tsunamayo** [comment]
Yes I am working on docking tech atm - I will add a smaller one, although not sure about the bricks sized one, it might be a block sized instead.
For smaller build I will also add a maglock.
For the docking port I will also add a finer grid placement, that s a valid point, I keep you posted. Thanks, good luck on your build!

**Comment by Kaiser-Indrasil** [comment]
Thank you for considering this! I love your work and dedication!

**Comment by ExodistSKY1** [comment]
Do you think you could make it to where we can place the docking ports on a child entity to get certain angles? or will this be more for a mag-lock?

**Comment by tsunamayo** [comment]
There is a problem for children entity, if you move the children it is supposed to propagate to the main one, which is not coded atm.
